src/compiler/commandLineParser.ts

```diff
@@ -373,6 +372,68 @@ namespace ts {
     }
 
     /**
+     * Parses non quoted strings separated by comma e.g. "a,b" would result in string array ["a", "b"]
+     * @param s
+     * @param existingValue
+     */
+    function parseMultiValueStringArray(s: string, existingValue: string[]) {

```

Comment:
I could use split but then I would need to iterate over result again to make sure the strings are non empty and are correct.